tests/8.LogicSig.ts

```diff
@@ -433,7 +434,7 @@ describe('signLogicSigTransaction', () => {
 
   function testSign(
     lsigObject:
-      | ReturnType<typeof algosdk.makeLogicSig>
+      | ReturnType<(program: Uint8Array, args?: Uint8Array[]) => LogicSig>
```

Comment:
`ReturnType<typeof algosdk.makeLogicSig>` was a way to get around the fact that `LogicSig` didn't used to be exported. Since it is now, I believe this can be replaced with `LogicSig` directly

Reply:
👍